src/app/containers/CpsRelatedContent/RelatedContentPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -351,10 +351,12 @@ exports[`RelatedContentPromo it renders a Story Promo wrapped in a Grid componen
           class="emotion-14 emotion-15"
         >
           <a
+            aria-labelledby="storyPromoLinkText-urn:bbc:ares::asset:pidgin/44508901"
```

Comment:
Should we have aria-labelledby here? I don't see more than 1 nested span with content?